test/test_interpolations.jl

```diff
@@ -98,18 +99,24 @@
         end
         @test all([all(0 .< i .<= n_basefuncs) for i ∈ Ferrite.celldof_interior_indices(interpolation)])
 
-    # Check for dirac delta property of interpolation
-    @testset "dirac delta property of dof $dof" for dof in 1:n_basefuncs
-        for k in 1:n_basefuncs
-            N_dof = shape_value(interpolation, coords[dof], k)
-            if k == dof
-                @test N_dof ≈ 1.0
-            else
-                factor = interpolation isa Lagrange{RefQuadrilateral, 3} ? 200 : 4
-                @test N_dof ≈ 0.0 atol = factor * eps(Float64)
+        # Check for evaluation type stability of interpolation
+        @testset "type stability dof $dof" for dof in 1:n_basefuncs
```

Comment:
Doesn't check for type-instabilities or?
```suggestion
        @testset "type correctness dof $dof" for dof in 1:n_basefuncs
```